### ISSUE TITLE
CircleOutlineGeometry appears as cylinder

### ISSUE THREAD

**Issue by gberaudo** [AnalyticalGraphicsInc/cesium#2499]
Hello,

The following Sandcastle code used to display a floating circle.
However in 1.6 and master, a cylinder appears.
This issue seems related to the packing changes.

```js
var viewer = new Cesium.Viewer('cesiumContainer');
var primitives = viewer.scene.primitives;

var center = new Cesium.Cartesian3(
    2930859.8012761218,
    2922254.5933887595,
    5476485.915141216
);


var radius = 648236;
var height = 500000;


var outlineGeometry = new Cesium.CircleOutlineGeometry({
  center: center,
  radius: radius,
  height: height
});

var outColor = { red: 1, green: 0, blue: 0, alpha: 1 };


var createColoredPrimitive = function(geometry, color) {
  var createInstance = function(geometry, color) {
    return new Cesium.GeometryInstance({
      // always update Cesium externs before adding a property
      geometry: geometry,
      attributes: {
        color: Cesium.ColorGeometryInstanceAttribute.fromColor(color)
      }
    });
  };

  var options = {
    // always update Cesium externs before adding a property
    flat: true // work with all geometries
  };

  var appearance = new Cesium.PerInstanceColorAppearance(options);

  var instances = createInstance(geometry, color);

  var primitive = new Cesium.Primitive({
    // always update Cesium externs before adding a property
    geometryInstances: instances,
    appearance: appearance
  });

  return primitive;
};

primitives.add(createColoredPrimitive(outlineGeometry, outColor));
```

**Comment by mramato** [comment]
Thanks for the report.  That's definitely a bug.  The workaround for now is to add `extrudedHeight : height` as a parameter to the geometry.  We'll definitely get this fixed for 1.7

**Comment by gberaudo** [comment]
Thanks @mramato for the workaround.
